### ISSUE TITLE
Comparing last two launches

### ISSUE THREAD

**Issue by priyankachaturvedi30** [reportportal/reportportal#1072]
**I want to compare last two launches**
...

**I see we have an option Comparing launches side by side - with the help of history table**
But what if my history depth exceeds 30. (max value) 
![image](https://user-images.githubusercontent.com/50797731/88051536-676d0980-cb76-11ea-8d02-558f565e0224.png)

I am facing this issue where my build execution run/number has exceeded 30, and now I am unable to compare the launches (Basically I have a requirement to compare my current launch with the previous launch). The problem can be easily solved if I have an option to compare my current launch with the previous/last launch that was executed.
Say for my Execution 20 -- > requires to be compared to Execution 19.
Execution 21 --> would be required to be compared to Execution 21 and so on.

This would give me a fair idea at the History level to basically compare the successive two launches at any point in time.

...

**Comment by evjlobanova** [comment]
@priyankachaturvedi30 But using History table you can compare 3  latest launches. Just choose the option History depth 3 and choose option Launches with the same name. 

Does it work for you?

**Comment by priyankachaturvedi30** [comment]
Yes, @evjlobanova . This helps. I got confused. But there is one usecase following the above I wanted to discuss further: Take a look at the one of my real time stats - 
![image](https://user-images.githubusercontent.com/50797731/88141377-2249e580-cc11-11ea-9b73-c0e4813c83f4.png)

Now my build 28 is the latest build having **17 failed cases** count 
When I select History depth as 3 - it shows the comparison for build 28, 27, and 26 (latest 3).
As you notice, build 27 has **21 failures** and 26 has **29 failures**
but the comparison is just on those 17 issue of last build.  shouldn't it be helpful if we select the maximum number out of 3 launches (or whatever value user select) and show history comparison on that. So in our case - comparison would be on all 29 (max number) failures, with green tiles in the build if the failure was not present in any of the builds. In that way user wont loose information about the issue that were there in last build but eventually got resolved in current one. This would give a clear picture of all the issues that we observed in history and make a viable history comparison. Just thinking out loud. Please suggest your thoughts.

**Comment by evjlobanova** [comment]
@priyankachaturvedi30 For now this logic can not be applied to the History table.

I can you propose to try to use a Most failed test case widget.

![image](https://user-images.githubusercontent.com/30413511/88152626-f462a200-cc0c-11ea-8086-f338515f1626.png)

![image](https://user-images.githubusercontent.com/30413511/88152554-da28c400-cc0c-11ea-9494-4130e89e3bfd.png)

I will not close this issue.
We will discuss this use case internally when we are designing a new version 6.

**Comment by priyankachaturvedi30** [comment]
Thanks for giving it a thought @evjlobanova . Much appreciated :)
